packages/dara-core/tests/js/template-root.spec.tsx

```diff
@@ -31,21 +31,21 @@ describe('TemplateRoot', () => {
         expect(getByText('Frame, Menu')).toBeInstanceOf(HTMLSpanElement);
     });
 
-    it('should clean up cache on startup', () => {
+    it('should clean up cache on startup', async () => {
```

Comment:
This test doesn't cover `sessionStorage` cleanup, could we have it cover both storages (or add another one to cover it)

Reply:
oops had forgotten to add sessionStorage, has been added now